app/helpers/layout_helper.rb

```diff
@@ -413,7 +400,8 @@ def icons(i)
   end
 
   def icon_text(i, text = "", opts = {})
-    (content_tag(:i,"", :class=>"glyphicon glyphicon-#{i} #{opts[:class]}", :title => opts[:title]) + " " + text).html_safe
+    opts[:kind] = "glyphicon glyphicon" if opts[:kind].nil?
```

Comment:
also, if its truly type / kind, it should be only glyphicon of pficon, no point in passing the string twice.